### PR TITLE
Feat/component/contextselector

### PR DESCRIPTION
**Pull request by eikeland**
# Component Development
Create a component for context selector using the searchable-dropdown web component.

### Type

- [x] Feature
- [ ] Fix
- [ ] Hotfix (should release ASAP)
- [ ] Maintenance (deps only)

#### Create User Story(DevOps) or Issue(Github)

- [ ] Describe functional requirements
- [ ] Design plan/suggestion (e.g. images, link to eds design or relevant figma links, mention of eds tokens to use)
- [ ] Communication with UX *(evaluate if needed)*

#### Development

- [ ] Describe the component
- [ ] Code funtional requirements
- [x] Create stories
- [ ] Create snapshots *(evaluate if needed)*
- [ ] Remove unused imports and commented code
- [ ] Lint code with EsLint (and follow best practises for [code quality](https://docs.fusion-dev.net/development/frontend/code-quality/)
  
#### Create Pull Request

- [ ] Assign relevant reviewers in Github and Cromatic
- [ ] Inform colleagues of pending reviews. We use Teams for this purpose, under fusion-frontend team please tag involved reviewers in the [Code review channel](https://teams.microsoft.com/l/channel/19%3a056a9ba8d8d84a058b24762f85c603ae%40thread.tacv2/Code%2520%2520review?groupId=9589ba16-68a5-47e6-97d7-7d0be0e0d3cb&tenantId=3aa4a235-b6e2-48d5-9195-7fcf05b459b0)

#### Review of Pull Request

- [ ] Follow up on comments from code review and implement necessary changes *(evaluate if needed)*
- [ ] Review approved in github (and chromatic)

#### Publish Code

- [ ] Increment version number
- [ ] Merge PR
- [ ] Consider letting fusion developers know this this shiny new thing exists

### CODE REVIEW

**Review by odinr — APPROVED** [review]

File: packages/context-selector/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export * from './Dropdown';
+export * from './DropdownProvider';
+export * from './ContextSelector';
+export * from './useDropdownProviderRef';
+
+export {
+  SearchableDropdownElement as HTMLSearchableDropdownCustomElement,
+  SearchableDropdownProviderElement as HTMLSearchableDropdownProviderCustomElement,
+} from '@equinor/fusion-wc-searchable-dropdown';
+
```
Comment (Noggling) [inline packages/context-selector/src/index.tsx:10]: Please expose all types needed
 SearchableDropdownResultItem, SearchableDropdownResult 

same shoud be done in @equinor/fusion-wc-searchable-dropdown

File: packages/context-selector/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "build": "tsc -b",
+    "lint": "eslint --ext .ts,.tsx,.js src/",
+    "lint:fix": "eslint --ext .ts,.tsx,.js src/ --fix",
+    "prepack": "npm run build"
+  },
+  "bugs": {
+    "url": "https://github.com/equinor/fusion-react-components/issues"
+  },
+  "dependencies": {
+    "@equinor/fusion-react-utils": "^2.0.3",
```
Comment (Noggling) [inline packages/context-selector/package.json:40]: please add `@equinor/fusion-react-styles` to dependencies or peerDependencies

**Review by Noggling — CHANGES_REQUESTED** [review]
Some refinement needed

File: packages/context-selector/src/index.tsx (hunk 1 above)
Comment (Noggling) [reply packages/context-selector/src/index.tsx:10]: sorry seems to be done in @equinor/fusion-wc-searchable-dropdown <3

**Pull request merged**